x-pack/plugins/lists/common/schemas/request/update_endpoint_list_item_schema.test.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { left } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+
+import { exactCheck, foldLeftRight, getPaths } from '../../siem_common_deps';
+
+import {
+  UpdateEndpointListItemSchema,
+  updateEndpointListItemSchema,
+} from './update_endpoint_list_item_schema';
+import { getUpdateEndpointListItemSchemaMock } from './update_endpoint_list_item_schema.mock';
+
+describe('update_exception_list_item_schema', () => {
```

Comment:
Nit - should this be `update_endpoint_list_item_schema`?

Reply:
Yeah will fix these now here.